sdv/metadata/multi_table.py

```diff
@@ -354,11 +370,32 @@ def detect_table_from_csv(self, table_name, filepath):
         """
         self._validate_table_not_detected(table_name)
         table = SingleTableMetadata()
-        data = table._load_data_from_csv(filepath)
-        table._detect_columns(data)
+        table.detect_from_csv(filepath)
```

Comment:
I think we need this change because `_load_data_from_csv` no longer exists in `SingleTableMetadata`. I added an integration test in [922aad0](https://github.com/sdv-dev/SDV/pull/1533/commits/922aad0898d17abab81f7b05f0231e5647658972) for the `detect_table_from_csv` which fails in the master branch

Reply:
It got moved here
https://github.com/sdv-dev/SDV/blob/387e9dd949abc0c412bd6e4b65e489cee1c93c08/sdv/utils.py#L168
I think we don't want to call `detect_from_csv` because if the error gets raised it says
https://github.com/sdv-dev/SDV/blob/387e9dd949abc0c412bd6e4b65e489cee1c93c08/sdv/metadata/single_table.py#L287-L289

Reply:
I see, thanks for the explanation, done in [ef79cf8](https://github.com/sdv-dev/SDV/pull/1533/commits/ef79cf86efbe222c76cd3b9c397240b0b94180cd)